cnxpublishing/views/publishing.py

```diff
@@ -266,4 +266,5 @@ def collate_content(request):
 WHERE uuid = %s
 """, (id,))
             publisher, message = cursor.fetchone()
-    collate(binder, publisher, message)
```

Comment:
The `@with_db_cursor` decorator supplies the cursor if it is not given as a keyword argument. Essentially this gives the functions the ability to  created their on connection or use an existing one. In this case, we moved it into the view's db cursor block to ensure it's part of the transaction that does the removal. Thus, if the re-collation fails, the previous collation will stay intact.